tutorials/2d/2d_movement.rst

```diff
@@ -315,10 +245,9 @@ on the screen will cause the player to move to the target location.
 
     public class Movement : CharacterBody2D
     {
-        [Export] public int speed = 200;
+        [Export] public int speed = 400;
 
         public Vector2 target = new Vector2();
```

Comment:
In GDScript the `target` field is initialized to `position`, since it's not possible to initialize to a non-static member in C# the closest equivalent would be to set this in the constructor or maybe `_Ready`. Not sure how important this is, it may not be worth adding if it distracts from the main goal of the tutorial.

Reply:
That was to prevent the sprite from moving to (0, 0) at first. It's not particularly important, and it wasn't there in the original version.